libc/string_cpy.c

```diff
@@ -0,0 +1,1269 @@
+/*
+ * Phoenix-RTOS
+ *
+ * POSIX.1-2017 standard library functions tests
+ * HEADER:
+ *		- string.h
+ * TESTED:
+ *		- memcpy()
+ *		- memccpy()
+ *		- strncpy()
+ *		- stpncpy()
+ *		- strcpy()
+ *		- stpcpy()
+ *		- strlcpy()
+ *		- strlcat()
+ *
+ * Copyright 2023 Phoenix Systems
+ * Author: Mateusz Niewiadomski, Damian Modzelewski
+ *
+ * This file is part of Phoenix-RTOS.
+ *
+ * %LICENSE%
+ */
+
+#include <stdio.h>
+#include <errno.h>
+#include <string.h>
+#include <limits.h>
+#include <stdlib.h>
+#include <sys/stat.h>
+#include <unistd.h>
+#include <unity_fixture.h>
+
+/* defines lengths of buffers holding strings*/
+#define MAX_STR_LEN   24
+#define MAX_ASCII_SET 256
+#define BIG_NUMB      1024
+
+/* defines for string and mem copy */
+#define TEST_STR1 "Lorem ipsum dolor"
+#define TEST_STR2 "Maecenas id commodo"
+#define TEST_STR3 "Hendrerit turpis"
+
+/* defines for string_strlcpy */
+#define STR_SRC  "abcd"
+#define STR_DEST "xxxx"
+
+/* defines for string_strlcat */
+#define STR_SRC1        "abc"
+#define STR_SRC2        "defgh"
+#define STR_PLACEHOLDER "klmnopqrstu"
+
+
+static char long_str[PATH_MAX];
+
+
+static int big_string(void)
+{
+	for (int i = 1; i < PATH_MAX; i++) {
+		long_str[i - 1] = i % MAX_ASCII_SET;
+	}
+	return 1;
```

Comment:
It's a common practice to return 0 as success
```suggestion
	return 0;
```